crates/sui-adapter/src/adapter.rs

```diff
@@ -644,6 +645,11 @@ pub fn resolve_and_type_check(
             })
         }
     };
+    if fdef.visibility != Visibility::Script {
```

Comment:
If you could add an extra condition here:
`&& ctx.digest() != TransactionDigest::genesis()`
would solve my problem of bypassing the visibility check during genesis.
Thanks!

Reply:
lol sure, why not